Birthday-2024-Project/Scripts/UI/UISoundPlayer.gd

```diff
@@ -29,6 +35,8 @@ func connect_buttons(root):
 
 func connect_to_button(button):
 	button.connect("pressed", _on_Button_pressed)
+	button.connect("mouse_entered", _on_Button_pressed)
```

Comment:
I think you intended this to be _on_Button_hovered

Reply:
So I was not mishearing things